src/main/java/at/hannibal2/skyhanni/test/TestExportTools.kt

```diff
@@ -52,8 +53,12 @@ object TestExportTools {
 
     @SubscribeEvent
     fun onKeybind(event: GuiScreenEvent.KeyboardInputEvent.Post) {
-        if (!config.copyItemDataCompressed.isKeyHeld() && !config.copyItemData.isKeyHeld()) return
+        if (noKeysHeld()) return
         val gui = event.gui as? GuiContainer ?: return
+        if (config.copyChestName.isKeyHeld()) {
+            copyChestName()
+            return
+        }
```

Comment:
At this point i would just add another event handler.

Reply:
@nea89o hanni finally got around to explaining what he meant by "line 61"

long story short we agreed to move all of this to a new `.kt` file instead. i'm gonna close this PR and move its features to the new one in a few minutes.